pkg/storage/tsdb/block/block_test.go

```diff
@@ -210,7 +210,7 @@ func TestUpload(t *testing.T) {
 		chunkFileSize := getFileSize(t, filepath.Join(tmpDir, b1.String(), ChunksDirname, "000001"))
 		require.Equal(t, chunkFileSize, int64(len(bkt.Objects()[path.Join(b1.String(), ChunksDirname, "000001")])))
 		require.Equal(t, 401, len(bkt.Objects()[path.Join(b1.String(), IndexFilename)]))
-		require.Equal(t, 570, len(bkt.Objects()[path.Join(b1.String(), MetaFilename)]))
+		require.Equal(t, 568, len(bkt.Objects()[path.Join(b1.String(), MetaFilename)]))
```

Comment:
Note to reviewers: it's 2 bytes smaller because previously we were serializing the resolution `124` while now we serialise the default resolution `0`.